packages/core/src/types.ts

```diff
@@ -1,9 +1,108 @@
 import * as CSS from 'csstype';
-import { UNSUPPORTED_CSS_PROPERTIES } from './constants';
 
 export type GriffelStylesCSSValue = string | 0;
 
-export type GriffelStylesUnsupportedCSSProperties = Record<keyof typeof UNSUPPORTED_CSS_PROPERTIES, never>;
+/**
+ * Griffel doesn't support expansion of CSS shorthands.
+ * They can be replaced either with the corresponding longhand properties or with the `shorthands` helper functions.
+ * @see https://griffel.js.org/react/guides/limitations#css-shorthands-are-not-supported
+ */
+export interface GriffelStylesUnsupportedCSSProperties extends Record<keyof CSS.StandardShorthandProperties, never> {
```

Comment:
```suggestion
export interface GriffelStylesUnsupportedCSSProperties extends Record<keyof typeof UNSUPPORTED_CSS_PROPERTIES, never> {
```

Is it still possible to use the internal unsupported properties lookup ? We do explicit handling there to strip those properties from the styles. It might seem stupid but at least we know that there is a link between what is happening in the library and the types here